mod.ts

```diff
@@ -0,0 +1,116 @@
+"use strict";
+
+const {
+  DBError,
+  UniqueViolationError,
+  NotNullViolationError,
+  ForeignKeyViolationError,
+  ConstraintViolationError,
+  CheckViolationError,
+  DataError,
+} = require("db-errors");
+const { Model: NativeModel } = require("./model/Model");
```

Comment:
You're not importing using the full filename (with extension, ie `./model/Model.ts`)
I'm pretty sure Deno is now capable of handling extensionless imports, but I cannot find where for now 🤔

Reply:
Ah sorry.. I didn't migrate anything in mod.ts 😆 I just renamed and that's it. The only part I've been migrating is src/utils/object.ts